PR_Epic0_13_Добавлены темы и стили приложения

A2T-user opened the pull request:
Сделано:
1. Дневная и ночная темы.Учтено изменение цвета строки состояния и navigationBar, а также контрастное выведение на них символов и иконок. 
2. Отдельные атрибуты для цетов меняющихся при смене день/ночь:
 2.1. Цвет фона
 2.2. Цвет основного текста
 2.3. Цвет текста подсказок для EditText
 2.4. Цвет фона для EditText
3. Добавлены фонты шрифтов. Сделаны стили для всех применяемых в приложении шрифтов (см. res\values\styles.xml)

A2T-user committed:
Сделано:
1. Дневная и ночная темы.Учтено изменение цвета строки состояния и navigationBar, а также контрастное выведение на них символов и иконок.
2. Отдельные атрибуты для цетов меняющихся при смене день/ночь:
 2.1. Цвет фона
 2.2. Цвет основного текста
 2.3. Цвет текста подсказок для EditText
 2.4. Цвет фона для EditText
3. Добавлены фонты шрифтов. Сделаны стили для всех применяемых в приложении шрифтов

A2T-user committed:
Merge branch 'refs/heads/develop' into Epic0_13_Theme

# Conflicts:
#	build.gradle.kts

sobolevkir commented on app/src/main/res/values/colors.xml:
```diff
@@ -1,5 +1,10 @@
+    <color name="red">#F56B6C</color>
+    <color name="gray">#AEAFB4</color>
+    <color name="light_gray">#E6E8EB</color>
+    <color name="background">#321A1B22</color>
```
Лучше написать название цвета, например dark_gray. У нас будет 3 оттенка серого в приложении.
Background - не очень подходящее название для цвета, хоть и написано в фигма-макете. 
В атрибутах вот уже backgroundColor - норм.

sobolevkir commented on app/src/main/res/values/themes.xml:
Можно добавить, чтобы статус бар ещё правильно окрашивался:
```
<item name="android:windowLightStatusBar">true</item>
<item name="android:windowLightNavigationBar">true</item>
```
Эти строки в файле themes.xml используются для управления цветом иконок и текста на статус-баре и навигационной панели в Android:

android:windowLightStatusBar:
Когда установлено в true, этот атрибут задаёт светлую тему для статус-бара. Это означает, что иконки и текст на статус-баре будут тёмными, что хорошо подходит для светлых фонов.
Если false, статус-бар будет тёмным, а иконки и текст — светлыми.
android:windowLightNavigationBar:

Когда установлено в true, навигационная панель (панель с кнопками «Назад», «Домой», и т.д.) будет светлой, а иконки — тёмными.
Если false, панель будет тёмной, а иконки — светлыми.

Также я бы добавил установку цветов таких элементов как checkbox, radiogroup и т.д.
Предлагаю изучить, как можно задать их общим кодом.
Например для Material - тем можно написать вот так:
```
<item name="colorControlActivated">@color/colorPrimary</item> <!-- Цвет активного состояния -->
<item name="colorControlNormal">@color/colorOnSurface</item> <!-- Цвет неактивного состояния -->
<item name="colorControlHighlight">@color/colorAccent</item> <!-- Цвет при взаимодействии (нажатии) -->
```
Эти атрибуты будут автоматически применяться к таким элементам, как CheckBox, Switch, RadioButton, и другим элементам UI с активными состояниями.

sobolevkir commented on app/src/main/res/values-night/themes.xml:
аналогично themes.xml

sobolevkir reviewed (changes requested):
Внёс несколько предложений

A2T-user committed:
Сделано:
1. Стили шрифтов вынесены в отдельный файл
2. Созданы сили CheckBox, RadioButton, EditText

sobolevkir commented on app/src/main/res/drawable/blue_cursor.xml:
Для чего задаёшь цвет курсора через drawable?
Почему не через свойство у _TextInputLayout_: `app:cursorColor="?attr/colorAccent"` ?
Нам везде подойдёт EditText ?

Просто в дизайне я видел такое:
<img width="288" alt="image" src="https://github.com/user-attachments/assets/6e9671d6-3faa-4f5e-81c6-d48af32faa3f">

Больше похоже на _TextInputLayout_...

sobolevkir commented on app/src/main/res/values/styles.xml:
Наименование стилей можно сделать просто: "CheckBox", "RadioButton", "EditText, Bold_32" и т.д.
